recipe/meta.yaml

```
@@ -47,11 +49,9 @@ about:
   license_file: LICENSE
   license_family: BSD
   doc_url: https://anaconda.cloud/
-  dev_url: https://anaconda.cloud/
+  dev_url: https://inspector.pypi.io/project/anaconda-cloud
```

```suggestion
  dev_url: https://anaconda.cloud/
```
inspector.pypi.io is just a visualization tool.

---

I couldn't find the repo to reference, but now that I have it, I will update it. Thank you!

---

It is better to leave anaconda.cloud, as our repo is private.